examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
The files `astra_*.py` in examples were actually not intended as examples but to check consistency between different modalities. Also this code invokes the helper functions to call the ASTRA projector, thus I'm going to remove these files from examples. However, I don't know where to put it and I would like to keep that code (including saving the images) since it is the quickest way to check consistency. So where can I it?